src/SQLite_Command.php

```diff
@@ -29,10 +29,6 @@ public function import( $args, $assoc_args ) {
 			WP_CLI::error( 'Please provide a file to import.' );
 		}
 
-		if ( ! Import::get_sqlite_plugin_version() ) {
-			WP_CLI::error( 'The SQLite integration plugin is not installed or activated.' );
-		}
```

Comment:
I realized the `SQLiteDatabaseIntegrationLoader::load_plugin()` method already performs an equivalent check. No need to duplicate that logic.